src/Schema/Builder/AbstractColumn.php

```diff
@@ -27,6 +28,8 @@
  *
  * Provides a fluent interface, which means that the methods can be chained together to create a column schema with
  * many properties in a single line of code.
+ *
+ * @deprecated Use {@see AbstractColumnSchema} instead. Will be removed in 2.0.0.
```

Comment:
Should `ColumnInterface` be marked as deprecated also?

Reply:
Yes, but this leads to psalm reporting errors in many places in the code. In order not to deal with suppressing these errors, I left it like this.